tools/WrapPchCompiler.sh

```diff
@@ -3,24 +3,21 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-flags=("$@")
-flags_to_use=()
-for i in "${!flags[@]}";
-do
-    if [ $i -eq 0 ]; then
-        flags_to_use+=("${flags[$i]}")
-        continue
-    fi
+compiler_invokation="$@"
```

Comment:
Use an array for this (`compiler_invokation=("$@")` and then `"${compiler_invokation[@]}"` later).  It handles quoting better.